pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementsLQubit.hpp

```diff
@@ -661,5 +714,21 @@ class Measurements final
         }
         return init_idx;
     }
+
+    std::unordered_map<std::string, ExpValFunc> expval_funcs_;
```

Comment:
Is this the best place to define this class member?

Any thoughts on how we could make this `const`, but also populate it with the entries of `init_expval_funcs_` ?

Reply:
With respect to location of this member within this class, perhaps it would be better to define it before the (private) member functions. Unless your point is about moving its definition to its parent class `MeasurementBase`? `const` keyword can be used to make this member constant, which then can only be initialized using constructor initializer list. In this case, `init_expval_funcs_` can be removed, and its body can be used directly in the initializer list. I have modified the code accordingly.